#! /bin/sh /usr/share/dpatch/dpatch-run
## cupsaccept.dpatch by Martin Pitt <mpitt@debian.org>
##
## DP: Allow to invoke accept command as cupsaccept/cupsreject, too, for less
## DP: namespace clobbering and consistency with disable/enable.
## DP: (http://www.cups.org/str.php?L2936)

@DPATCH@
diff -urNad trunk~/systemv/accept.c trunk/systemv/accept.c
--- trunk~/systemv/accept.c	2008-09-06 13:51:16.000000000 +0200
+++ trunk/systemv/accept.c	2008-09-06 13:56:08.000000000 +0200
@@ -62,9 +62,9 @@
 
   cancel = 0;
 
-  if (!strcmp(command, "accept"))
+  if (!strcmp(command, "cupsaccept") || !strcmp(command, "accept"))
     op = CUPS_ACCEPT_JOBS;
-  else if (!strcmp(command, "reject"))
+  else if (!strcmp(command, "cupsreject") || !strcmp(command, "reject"))
     op = CUPS_REJECT_JOBS;
   else if (!strcmp(command, "cupsdisable") || !strcmp(command, "disable"))
     op = IPP_PAUSE_PRINTER;
